### PR TITLE
Add/fvm introduction

### PR DESCRIPTION
**Pull request by johnnymatthews**
Adds a easy-to-read introduction to the FVM, what it is, what it's used for, and why it's being built now. Also adds a placeholder for a _How it works_ page.

### CODE REVIEW

File: content/en/fvm/basics/how-it-works/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "How it works"
+description: "The FVM is a fairly complex technology, with a lot of moving part. On this page we'll cover the very basics of how the FVM functions, and point you to more indepth docs should you want to learn more."
```
Comment (aashidham) [inline content/en/fvm/basics/how-it-works/index.md:3]: moving parts

File: content/en/fvm/basics/introduction/index.md [hunk 2]
```diff
@@ -12,6 +12,52 @@ aliases:
...
 {{< beta-warning >}}
 
-For the best intro to the FVM, check out this presentation from FIL Lisbon 2022:
+## What's it for
+
+The FVM enables developers to build many new use cases on top of and with the Filecoin network.
+
+### Data access control
+
+Centralized storage solutions, like AWS or Google Cloud Services, have one substantial advantage over web3 services: they can limit individual data access.
```
[the remark this answers is not in the document]
Comment (johnnymatthews) [reply content/en/fvm/basics/introduction/index.md]: Cleaned up that statement a little bit.

File: content/en/fvm/basics/introduction/index.md [hunk 3]
```diff
@@ -12,6 +12,50 @@ aliases:
...
-For the best intro to the FVM, check out this presentation from FIL Lisbon 2022:
+## What's it for
+
+The FVM enables developers to build many new use cases on top of and with the Filecoin network.
+
+### Data access control
+
+One advantage centralized storage solutions have over web3 services is that they can limit individual data access.
+
+The FVM enables Filecoin network participants to do the same. Threshold encryption networks, like Medusa, can individually or programmatically govern who can access a data collection without any contracts or validators having access to the data themselves.
```
Comment (johnnymatthews) [inline content/en/fvm/basics/introduction/index.md:23]: @aashidham, I reduced the _aggressiveness_ of the first sentence here. I also removed the last sentence that reinforces the benefits of FVM + data access control. What're your thoughts here?

**Review by aashidham — APPROVED** [review]

**Comment by johnnymatthews** [comment]
Closed in https://github.com/filecoin-project/filecoin-docs/pull/1484

**Pull request closed without merging**